scripts/testserver.js

```diff
@@ -12,6 +13,8 @@ const yargsParser = require('yargs-parser');
 const { rimraf } = require('rimraf');
 const glob = require('glob');
 
+dns.setDefaultResultOrder('ipv4first');
```

Comment:
`// TODO(COMPASS-6516)` everywhere?

Reply:
Yes, but right now I'm wondering if this is more of an investigation PR :) At least windows is green with these added.